src/collectors/windows.plugin/metadata.yaml

```diff
@@ -0,0 +1,265 @@
+plugin_name: windows.plugin
+modules:
+  - meta:
+      plugin_name: windows.plugin
+      module_name: PerflibProcesses
+      monitored_instance:
+        name: System statistics
+        link: "https://www.microsoft.com/en-us/windows"
+        categories:
+          - data-collection.windows-systems
+        icon_filename: "windows.svg"
+      related_resources:
+        integrations:
+          list: []
+      info_provided_to_referring_integrations:
+        description: ""
+      keywords:
+        - process counts
+        - threads
+        - context switch
+      most_popular: false
+    overview:
+      data_collection:
+          metrics_description: |
+            Perflib provides different statistical methods about Microsoft Windows environment. This collector query for
+            Object 'System' to show actual number of processes, threads and context switches.
+          method_description: ""
+      supported_platforms:
+        include: ["windows"]
+        exclude: []
+      multi_instance: false
+      additional_permissions:
+        description: ""
+      default_behavior:
+          auto_detection:
+            description: |
+              The collector auto-detects all metrics. No configuration is needed.
+          limits:
+            description: ""
+          performance_impact:
+            description: ""
+    setup:
+      prerequisites:
+        list: []
+      configuration:
+          file:
+            section_name: "plugin:windows"
+            name: "netdata.conf"
+            description: ""
+          options:
+            description: ""
+            folding:
+              title: ""
+              enabled: true
+            list: []
+          examples:
+            folding:
+              enabled: true
+              title: ""
+            list: []
```

Comment:
can you check how the other collectors that use netdata.conf as a conf file word it?

https://github.com/netdata/netdata/blob/282836b8ef99ac51eabe1a3b13c5945eb6b43968/src/collectors/tc.plugin/metadata.yaml#L47-L97

In your generated js file you can read

```
Configuration\n\n#### File\n\nThe configuration file name for this integration is `netdata.conf`.\nConfiguration for this specific integration is located in the `plugin:windows` section within that file.\n\nThe file format is a modified INI syntax. The general structure is:\n\n```ini\n[section1]\n option1 = some value\n option2 = some other value\n\n[section2]\n option3 = some third value\n```\nYou can edit the configuration file using the `edit-config` script from the\nNetdata [config directory](/docs/netdata-agent/configuration/README.md#the-netdata-config-directory).\n\n```bash\ncd /etc/netdata 2>/dev/null || cd /opt/netdata/etc/netdata\nsudo ./edit-config netdata.conf\n```\n#### Options\n\n\n\nThere are no configuration options.\n\n#### Examples\nThere are no configuration examples.\n\n",
```

Having a configuration file and not giving configuration options and examples results in something looking completely wrong 

```
There are no configuration options.\n\n#### Examples\nThere are no configuration examples.
```

While at the start

```
The configuration file name for this integration is `netdata.conf`.\nConfiguration for this specific integration is located in the `plugin:windows` section within that file.\n\nThe file format is a modified INI syntax. The general structure is:\n\n```ini\n[section1]\n option1 = some value\n option2 = some other value\n\n[section2]\n option3 = some third value\n```\nYou can edit the configuration file using the `edit-config` script from the\nNetdata [config directory](/docs/netdata-agent/configuration/README.md#the-netdata-config-directory).
```

please also format the file to fix the inconsistencies with the indentation.